website/blog/2023-05-02-release-0.15.md

```diff
@@ -5,6 +5,7 @@ slug: podman-desktop-release-0.15
 authors: [deboer]
 tags: [podman-desktop, release, kubernetes, kind]
 hide_table_of_contents: false
+image: /assets/images/podman-desktop-release-0.15-89cd5f136ad85c15bc5b4d16a919140a.webp
```

Comment:
maybe we need to move images to static folder so you can use the same name. I'm not sure how the suffix is generated here so I don't know if it can be broken after an update ?

Reply:
So I optimized them and put them in a static folder (will push to PR after finishing this comment)... and I have no idea why but if I visit http://localhost:3000/img/blog/podman-desktop-release-1.3.webp in my browser, the image works no problem. It doesn't if I set the header in the blog post image attribute like so:

`image: /img/blog/podman-desktop-release-1.3.webp`

It won't display in the social preview:
![Screenshot from 2023-08-27 21-53-38](https://github.com/containers/podman-desktop/assets/799683/dc5f2a49-059f-4be8-8f72-8bde68302788)

If I hardcode it including http (https does not work) like so, it works:

`image: http://localhost:3000/img/blog/podman-desktop-release-1.3.webp`

![Screenshot from 2023-08-27 21-53-25](https://github.com/containers/podman-desktop/assets/799683/b4231625-b884-4410-a39b-aee42f4d1e14)

Docusaurus is doing something weird to relative links... any ideas?

Reply:
AFAIK it looks like the relative `image` tag in the blog post are prefixed by the docusaurus URL object

I've created https://github.com/containers/podman-desktop/pull/3685

Reply:
It does!!! Thanks @benoitf !!